community-containers/memories/memories.json

```diff
@@ -0,0 +1,33 @@
+{
+    "aio_services_v1": [
+        {
+            "container_name": "nextcloud-aio-memories",
+            "display_name": "Transcoding container for memories",
```

Comment:
Can we change this to "Memories Transcoder"? It's looking weirdly out of place in the UI.

Reply:
done